components/MerkleTree/Container/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Visualisation from "../Visualisation";
+import { Input } from "@chakra-ui/react";
+import { useState } from "react";
+import { generateTree } from "./utils";
+
+const Container = () => {
+    const [input, setInput] = useState("")
+
+    const handleChange = (e: any) => {
```

Comment:
Instead of any try using `React.ChangeEvent<HTMLInputElement>` in place of `any` here.

Reply:
This is fixed